lib/misc_funcs.sh

```diff
@@ -61,7 +61,7 @@ function export_mix_env() {
     if [ -d $env_path ] && [ -f $env_path/MIX_ENV ]; then
       export MIX_ENV=$(cat $env_path/MIX_ENV)
     else
-      export MIX_ENV=prod
+      export MIX_ENV=${1:-prod}
```

Comment:
Sorry, I am not very familiar with bash syntax. How does this ensure we build in the `test` environment?